src/rsa/rsa.rs

```diff
@@ -67,11 +67,23 @@ macro_rules! rsa_pkcs1_padding {
         pub static $PADDING_ALGORITHM: padding::PKCS1 =
             padding::PKCS1 {
                 digest_alg: $digest_alg,
-                digestinfo_prefix: $digestinfo_prefix,
+                digestinfo_prefix: $digestinfo_prefix
         };
     }
 }
 
+macro_rules! rsa_pss_padding {
+    ( $PADDING_ALGORITHM:ident, $digest_alg:expr, $doc_str:expr ) => {
+        #[doc=$doc_str]
+        /// Feature: `rsa_signing`.
+        pub static $PADDING_ALGORITHM: padding::PSS =
+            padding::PSS {
+                digest_alg: $digest_alg
```

Comment:
Not sure what style we have for this, but in this case I would just put the whole declaration of `padding::PSS { }` on a single line.